tests/model_registry/conftest.py

```diff
@@ -177,15 +177,23 @@ def model_registry_instance(
 
 @pytest.fixture(scope="class")
 def model_registry_mysql_config(
-    model_registry_db_deployment: Deployment, model_registry_db_secret: Secret
+    request: FixtureRequest,
+    model_registry_db_deployment: Deployment,
+    model_registry_db_secret: Secret,
 ) -> dict[str, Any]:
+    """
+    Fixture to build the MySQL config dictionary for Model Registry.
+    Expects request.param to be a dict. If 'ssl_ca' is not present, it defaults to None.
+    """
+    param = request.param if hasattr(request, "param") else {}
     return {
         "host": f"{model_registry_db_deployment.name}.{model_registry_db_deployment.namespace}.svc.cluster.local",
         "database": model_registry_db_secret.string_data["database-name"],
         "passwordSecret": {"key": "database-password", "name": model_registry_db_deployment.name},
-        "port": 3306,
+        "port": param.get("port", 3306),
         "skipDBCreation": False,
         "username": model_registry_db_secret.string_data["database-user"],
+        "ssl_ca": param.get("ssl_ca"),
```

Comment:
won't this fail if you don't pass in ssl_ca?

Reply:
and what happens to the mysql config if it has `ssl_ca` defined None/empty? I'm afraid that's what would break in the MR instance down the line

Reply:
you are right if it is not passed the attribute should not be present at all in the config